sematic/ui/src/types/Types.tsx

```diff
@@ -825,6 +825,17 @@ function S3Button(props: {region?: string, bucket: string, location?: string}) {
   </>;
 }
 
+function ImageValueView(props: ValueViewProps) {
+  const { valueSummary } = props;
+  const { bytes } = valueSummary;
```

Comment:
Maybe there is room to improve the typing of `valueSummary`, please kindly check if it makes sense.